Docs: Add NGINX example for using websockets to Loki (#27998)
grafana/grafana: docs/sources/features/datasources/loki.md

```
ProxyPassMatch "^/(api/datasources/proxy/\d+/loki/api/v1/tail)" "ws://127.0.0.1:3000/$1"
```

> Example below shows basic NGINX proxy configuration. Example assumes that Grafana server is available at `http://localhost:3000/`, Loki server is running locally without proxy and your external site uses HTTPS. If you also host Loki behind nginx proxy, you might want to repeat following configuration for Lokis as well.

Reviewer: Please capitalize NGINX consistently.

> Example below shows basic NGINX proxy configuration. Example assumes that Grafana server is available at `http://localhost:3000/`, Loki server is running locally without proxy and your external site uses HTTPS. If you also host Loki behind nginx proxy, you might want to repeat following configuration for Lokis as well.

Reviewer: ```suggestion
Example below shows basic NGINX proxy configuration. The example assumes that the Grafana server is available at `http://localhost:3000/`, Loki server is running locally without proxy, and your external site uses HTTPS. If you also host Loki behind nginx proxy, then you might want to repeat the following configuration for Loki as well.
```

In `http` section of NGINX add following map definition:
```
  map $http_upgrade $connection_upgrade {
    default upgrade;
    '' close;
  }
```

> In your `server` section add following configuration:

Reviewer: ```suggestion
In your `server` section, add following configuration:
```

Author: Are you sure about the comma? What about `the` before `following`

Reviewer: Good point. Added.

```
  location ~ /(api/datasources/proxy/\d+/loki/api/v1/tail) {
      proxy_pass          http://localhost:3000$request_uri;
      proxy_set_header    Host              $host;
      proxy_set_header    X-Real-IP         $remote_addr;
      proxy_set_header    X-Forwarded-for   $proxy_add_x_forwarded_for;
      proxy_set_header    X-Forwarded-Proto "https";
      proxy_set_header    Connection        $connection_upgrade;
      proxy_set_header    Upgrade           $http_upgrade;
  }

  location / {
      proxy_pass          http://localhost:3000/;
      proxy_set_header    Host              $host;
      proxy_set_header    X-Real-IP         $remote_addr;
      proxy_set_header    X-Forwarded-for   $proxy_add_x_forwarded_for;
      proxy_set_header    X-Forwarded-Proto "https";
  }
```

> > **Note:** This feature is only available in Grafana v6.3+

Reviewer: ```suggestion
> **Note:** This feature is only available in Grafana v6.3+.
```

```
ProxyPassMatch "^/(api/datasources/proxy/\d+/loki/api/v1/tail)" "ws://127.0.0.1:3000/$1"
```

> Example below shows basic NGINX proxy configuration. This example assumes that the Grafana server is available at `http://localhost:3000/`, Loki server is running locally without proxy, and your external site uses HTTPS. If you also host Loki behind NGINX proxy, then you might want to repeat the following configuration for Loki as well.

Reviewer 2: The following example below shows the basic NGINX proxy configuration. It assumes that the Grafana server is available at `http://localhost:3000/`, Loki server is running locally without proxy, and your external site uses HTTPS. If you also host Loki behind NGINX proxy, then you might want to repeat the following configuration for Loki as well.

> In `http` section of NGINX configuration, add the following map definition:

Reviewer 2: In the `http` section of NGINX configuration, add the following map definition:

Author: Done

```
  map $http_upgrade $connection_upgrade {
    default upgrade;
    '' close;
  }
```
In your `server` section add following configuration:
```
  location ~ /(api/datasources/proxy/\d+/loki/api/v1/tail) {
      proxy_pass          http://localhost:3000$request_uri;
      proxy_set_header    Host              $host;
      proxy_set_header    X-Real-IP         $remote_addr;
      proxy_set_header    X-Forwarded-for   $proxy_add_x_forwarded_for;
      proxy_set_header    X-Forwarded-Proto "https";
      proxy_set_header    Connection        $connection_upgrade;
      proxy_set_header    Upgrade           $http_upgrade;
  }

  location / {
      proxy_pass          http://localhost:3000/;
      proxy_set_header    Host              $host;
      proxy_set_header    X-Real-IP         $remote_addr;
      proxy_set_header    X-Forwarded-for   $proxy_add_x_forwarded_for;
      proxy_set_header    X-Forwarded-Proto "https";
  }
```

> > **Note:** This feature is only available in Grafana v6.3+.

Reviewer: This sentence needs to be before the feature description, not after.

Author: This is not my change, but looking at the whole file, it's always at the end of documentation snippet.

Author: What about this note?
